tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py

```diff
@@ -3,14 +3,25 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from marionette.by import By
-from gaiatest.apps.base import Base
+from gaiatest.apps.base import PageRegion
 
 
-class StatusBar(Base):
+class StatusBar(PageRegion):
+    _status_bar_locator = (By.ID, 'statusbar')
     _status_bar_time_locator = (By.ID, 'statusbar-time')
     _status_bar_maximized_wrapper = (By.ID, 'statusbar-maximized-wrapper')
     _status_bar_minimized_wrapper = (By.ID, 'statusbar-minimized-wrapper')
 
+    def __init__(self, marionette):
+        marionette.switch_to_frame()
+        status_bar = marionette.find_element(*self._status_bar_locator)
+        PageRegion.__init__(self, marionette, status_bar)
+
+    @property
+    def is_status_bar_visible(self):
```

Comment:
I would rename this property to `is_displayed` in order to avoid the repetition of `status_bar` and to be consistent with Marionette.